examples/userdemo/colormap_normalizations_diverging.py

```diff
@@ -0,0 +1,44 @@
+"""
+=====================================
+Diverging Norm Colormap Normalization
+=====================================
+
+Sometimes we want to have a different colormap on either side of a
+conceptual center point, and we want those two colormaps to have
+different linear scales.  An example is a topographic map where the land
+and ocean have a center at zero, but land typically has a greater
+elevation range than the water has depth range, and they are often
+represented by a different colormap.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.cbook as cbook
+import matplotlib.colors as colors
+
+filename = cbook.get_sample_data('exampleTopo.npz', asfileobj=False)
```

Comment:
rename it example_topo.npz, or perhaps just topography.npz (it's obviously an example...) or vancouver.npz :)

Reply:
Changed to `topobathy.npz` just to make clear it has ocean data too...